park-dashboard/src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core/styles";
+import HomeIcon from "@material-ui/icons/Home";
+import ParkIcon from '@mui/icons-material/Park';
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  title: {
+    flexGrow: 1,
+  },
+  parklogo: {
+    color: "green",
+    marginRight: theme.spacing(1),
+  },
+  navbarButton: {
+    height: "100%",
+    display: "block",
+    padding: "20px",
+    alignItems: "center",
+    "&:hover": {
+      backgroundColor: "#333333",
+      color: "white",
+    },
+    "&:active": {
+      backgroundColor: "#faa810",
+    },
+  },
+  navbarLink: {
+    height: "100%",
+    font: "bold",
+    display: "block",
+    color: theme.palette.common.white,
+    textDecoration: "none",
+  },
+  header: {
+    position: "fixed",
+    backgroundColor: "black",
+    color: "White",
+    boxShadow: "1px 0px 0px 0px",
+  },
+}));
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: "#FF000",
+    },
+  },
+});
+const Navbar = () => {
+  const classes = useStyles();
+  return (
+    <ThemeProvider theme={theme}>
+      <div className={classes.root}>
+        <AppBar className={classes.header}>
+          <Toolbar>
+            <Typography variant="h6" className={classes.title}>
+              Park Dashboard
+              <ParkIcon className={classes.parklogo} />
+            </Typography>
+            <Button color="inherit" className={classes.navbarButton}>
+              <Link to="/" className={classes.navbarLink}>
+                <HomeIcon className={classes.logo} />
```

Comment:
The home icon button seems redundant maybe you can add return to home link to the logo. Or else we can add label to the button so that we don't get accessibility errors.

Reply:
Remove the HomeIcon and add the home link to logo.